src/hotspot/cpu/riscv/riscv_v.ad

```diff
@@ -76,6 +76,15 @@ source %{
       case Op_VectorCastHF2F:
       case Op_VectorCastF2HF:
         return UseZvfh;
+      case Op_VectorLoadShuffle:
+      case Op_VectorRearrange:
+        // vlen >= 4 is required, because min vector size for byte is 4 on riscv,
```

Comment:
I was not aware of such a constraint before. Is this a constraint at the ISA level or is a performance consideration? I didn't find where it is mentioned in the RVV spec.

Reply:
No, it's an existing constraint in jdk itself, please check the code in `Matcher::min_vector_size`, I think it's partially for performance consideration. It's the same as aarch64 and x86.